core/mutation/mutation.go

```diff
@@ -28,6 +28,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 
+	spb "github.com/google/keytransparency/impl/proto/mutation_v1_service"
```

Comment:
I don't think this will work. `core/mutation` cannot depend on `impl/proto/mutation_v1_service`.
If I remember correctly, the entire implementation of the streaming API should be done in `impl/proto/mutation_v1_service`. We might need to refactor `core/mutation/mutation.go` a little bit to reuse some of the core functionality.